CodingChallenges/CC_128_SketchRNN_Snowflakes/P5/sketch.js

```diff
@@ -8,12 +8,11 @@ let model;
 let strokePath = null;
 
 let x, y;
-let pen = "down";
+let pen;
 
 function setup() {
   createCanvas(windowWidth, windowHeight);
-  x = random(-width / 2, width / 2);
-  y = random(-height / 2, height / 2);
+  setupNewSketch();
```

Comment:
Would you mind adding some comments explaining what's changed here from the video? If you prefer I can add them.

Reply:
@shiffman I've done so in the last commit. I'm not sure if that's what you wanted me to do, and also I'm not a native English speaker, so feel free to revert and add the comments yourself if you want to :)